includes/admin.form.inc

```diff
@@ -39,6 +39,13 @@ function islandora_basic_collection_admin(array $form, array &$form_state) {
           '#default_value' => variable_get('islandora_basic_collection_page_size', '12'),
           '#description' => t('The default number of objects to show in a collection view.'),
         ),
+        'islandora_basic_collection_admin_page_size' => array(
```

Comment:
@rosiel

This item is for the management interface not the object display and should be moved out of the state controlled fieldset as it is not effected by the `Completely disable default collection display generation. ` setting.

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.](http://www.discoverygarden.ca) | Managing Digital Content*